src/02_spec/07_modules/dem_uart/index.rst

```diff
@@ -0,0 +1,36 @@
+UART Device Emulation Module (DEM-UART)
+=======================================
+
+.. figure:: img/osd_dem_uart.*
+   :alt: Device Emulation Module UART
+
+The UART Device Emulation Module connects to the bus of a given CPU on one side,
+and to the Debug Interconnect on the other.
+Towards the CPU it behaves like a UART-16550A device, but instead of transmitting information
+over a UART-Interface it instead passes it to a host PC via the Debug Interconnect.
+
+The following UART registers are implemented and accessible via the bus, the address mapping
+is according to the `UART 16550 Datasheet <http://caro.su/msx/ocm_de1/16550.pdf>`_.
+The implemented properties are specified in the `OpenRISC Device Tree <https://elixir.bootlin.com/linux/latest/source/arch/openrisc/boot/dts/or1ksim.dts>`_,
+See `this <https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/serial/8250.txt>`_ for more information about the Device Tree specification.
+No Modem or DMA-Mode related features, registers or interrupts are implemented.
+
+.. _uart_registers:
+
+16550 UART Registers
+--------------------
+All registers are 8 bit wide.
+
++-----------+-------------+-------------------------------------+
+| Address   | Register    | Description                         |
++===========+=============+=====================================+
+| ``0x00``  | ``RBR/THR`` | Receiver/Transmitter Buffer         |
++-----------+-------------+-------------------------------------+
+| ``0x01``  | ``IER``     | Enable/Disable certain Interrupts   |
++-----------+-------------+-------------------------------------+
+| ``0x02``  | ``IIR/FCR`` | Qualify Interrupt and control FIFOs |
++-----------+-------------+-------------------------------------+
+| ``0x03``  | ``LCR``     | Access to the DLAB                  |
++-----------+-------------+-------------------------------------+
+| ``0x05``  | ``LSR``     | Information about state of the UART |
++-----------+-------------+-------------------------------------+
```

Comment:
For all of this table:
- You can use the `flat-table` rst feature to make it easier to write tabular content.
- Please use consistent upper/lowercase spelling.
- Please be specific. For example: "Enable/Disable certain Interrupts" raises two questions: which interrupts are "certain interrupts"? Does writing a 1 enable interrupts, or are they enabled by writing a 0? I know that all these registers are defined in other specifications already. If that is the case, link to this specification close to where you explain them.